app/assets/javascripts/discourse/app/helpers/directory-item-helpers.js

```diff
@@ -0,0 +1,37 @@
+import { htmlSafe } from "@ember/template";
+import { number } from "discourse/lib/formatter";
+import { registerUnbound } from "discourse-common/lib/helpers";
+import I18n from "I18n";
+
+registerUnbound("mobile-directory-item-label", function (args) {
```

Comment:
There are too many helpers for `directory-items`. I decided to shove all the related helpers into 1 file just to be easier when developing.